### ISSUE TITLE
Registration Number for domiciliary Care

### ISSUE THREAD

**Issue by athulmanuel** [coronasafe/care_fe#9513]
<img width="791" alt="Image" src="https://github.com/user-attachments/assets/0fdc2509-af28-450b-b6a1-49d39496fa3b" />

Could we change the term from OP number to registration number ?

**Comment by modamaan** [comment]
@Jacobjeevan i have updated, could you assign it to me ?

**Comment by Jacobjeevan** [comment]
> [@Jacobjeevan](https://github.com/Jacobjeevan) i have updated, could you assign it to me ?

Don't work on issues that weren't assigned to you 👍 All the new issues are blocked and assigned to core team as they are reliant on changes in form-field-branch.

**Comment by bodhish** [comment]
@athulmanuel can you re-test the form!

**Comment by bodhish** [comment]
@athulmanuel do we have a Registration Number for domiciliary Care? If not we can hide it!

**Comment by athulmanuel** [comment]
yes, till now there was a registry that is being kept as Nominal register with a continuous numbering for al registration. once we fully digitise the system we can remove it. 

To start with we will need it. 


Will have to figure out a system as all the facilites that we enter will be running facility and they will have a system to register patient and keep track of the number, we must have a strategy on how to transition it fully into CARE by just keeping the count of patient till now.

**Comment by bodhish** [comment]
Resolved
